electrification_signals.mss

```diff
@@ -118,6 +118,54 @@
         marker-allow-overlap: true;
       }
     }
+
+    /************************************/
+    /* DE begin of isolated overlap     */
+    /************************************/
+    ["electricity_type"="begin_of_isolated_overlap"]["electricity_form"="sign"] {
```

Comment:
I wonder if this should be` ^(.*;)?begin_of_isolated_overlap(;.*)?$` because in many cases the beginning and end sign will be in the same node:

```
railway=signal
railway:signal:electricity=DE-ESO:streckentrennung
railway:signal:electricity:form=sign
railway:signal:electricity:type=begin_of_isolated_overlap;end_of_isolated_overlap
```

Reply:
May be true but I havent seen any where begin/end of the same direction are on the same pole and you'd need two nodes for the two different directions. 
Yet if it would be possible, it could be beneficial to have different "layering" values?

Reply:
If I've understood Carto correct, there is no layering; and I havent seen any same-direction-poles with both begin and end, only with different directions (at least in Germany); so I would like to keep that as is for the moment.